controls/radchartview/series/cartesianchart-series/area-series/stepareaseries.md

```diff
@@ -40,7 +45,66 @@ You can use the following definition to display a simple StepAreaSeries:
 	</telerik:RadCartesianChart.Series>
 	</telerik:RadCartesianChart>
 {{endregion}}
-![radchartview-series-steplineareaseries](images/radchartview-series-steplineareaseries.png)
 
-# See Also
- * [Chart Series Overview]({%slug radchartview-series-chartseries%})
\ No newline at end of file
+#### __Figure 1: StepAreaSeries visual appearance__
+![radchartview-series-stepareaseries](images/radchartview-series-steplineareaseries.png)
+
+## Properties
+
+* __CategoryBinding__: A property of type __DataPointBinding__ that gets or sets the property path that determines the category value of the data point.
+* __ValueBinding__: A property of type __DataPointBinding__ that gets or sets the property path that determines the value of the data point.
+* __Fill__: A property of type __Brush__ that gets or sets the color of the StepAreaSeries area.
+* __DashArray__: A property of type __DoubleCollection__ that gets or sets the dash pattern applied to the stroke of the area.
+* __Stroke__: A property of type __Brush__ that gets or sets the outline stroke of the StepAreaSeries area.
+* __AreaShapeStyle__: A property of type __Style__ that gets or sets the appearance of the area shape. The property excepts a style that targets an object of type Path.
+* __StrokeShapeStyle__: A property of type __Style__ that gets or sets the style of the stroke of the area shape. The property accepts a style that targets an object of type Path.
+* __OriginValue__: A property of type double which controls from which Y point value the area series should be start drawing.

```

Comment:
This is not true. The property doesn't control from which Y point the value of the area should start drawing. The value could be also a X point if the numeric axis is set as HorizontalAxis of the chart. 

The property controls the origin value from which the series should start drawing on the numeric axis (LinearAxis or LogarithmicAxis).

Update this information on all places where the OriginValue of the series is described.